### PR TITLE
Necropolis Tendrils will produce an outer ring of lava when collapsing

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan
tweak: Necropolis Tendrils will produce an outer ring of lava when collapsing.
/:cl:

This gives you a slight delay to rethink your life choices when you're about to run into a chasm.
Also, it glows, so you can see it.
Closes #27526

### CODE REVIEW

File: code/modules/mob/living/simple_animal/hostile/mining_mobs.dm [hunk 1]
```diff
@@ -1031,25 +1031,37 @@
...
-			shake_camera(M, 15, 1)
-		playsound(get_turf(src),'sound/effects/explosionfar.ogg', 200, 1)
-		visible_message("<span class='boldannounce'>The tendril falls inward, the ground around it widening into a yawning chasm!</span>")
-		for(var/turf/T in range(2,src))
-			if(!T.density)
-				T.TerraformTurf(/turf/open/chasm/straight_down/lava_land_surface)
-		qdel(src)
+/obj/effect/collapse/Initialize()
+	. = ..()
+	var/turf/T = get_turf(src)
```
Comment (Cyberboss) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs.dm]: For both cases, why use the turf instead of the obj?

**Comment by Repukan** [comment]
Hugbox 👎👎

**Comment by Steelpoint** [comment]
ez mode.

Seriously why all this protection for people falling into chasms?

**Comment by obscolene** [comment]
because chasms are not fun. they add nothing of value to the game, you just fall into them and are permanently removed from the round.

**Comment by Repukan** [comment]
Look where you're walking you aspie

**Comment by ChangelingRain** [comment]
Chasms wouldn't be as awful if they dropped you to a lower layer but right now they're just instakills for not being 100% careful and that's just not fun.

**Comment by Repukan** [comment]
I mean many things can fuck you up too if you just run with no situational awareness.

**Comment by ChangelingRain** [comment]
Yeah but none of them are instant; watchers slow you, goliaths stun and give you a second or two to watch yourself die, lava slows you and takes a bit to kill if you don't roll, legions just aren't that fast at kills

I guess the colossus could oneshot you if you bumped right into it and it shotgunned you, but that's basically it

**Review by Cyberboss — APPROVED** [review]

**Comment by Repukan** [comment]
We don't need to baby proof the station, much less lavaland.

We have threads on this on the forum that outline many reasons why this is a less than desirable idea. I dislike that it will set a precedent for future "QoL" terrain changes.

**Comment by ChangelingRain** [comment]
Instant death isn't fun! A protracted, slow, unavoidable death is much more interesting.

**Comment by Repukan** [comment]
No it isn't.

You already know you're gonna die, what's the point of making it slower?

**Comment by GunHog** [comment]
I believe this change is to support the nerf on mesons, which would nearly blind most miners due to the combination of darkness, limited vision range, and small color difference between normal floor and chasms. If the meson nerf were (hopefully) closed, this would not be necessary.

**Comment by Repukan** [comment]
Then make the chasms glow a little, not put a ring around it

**Comment by Steelpoint** [comment]
Isn't there already a PR to make chasms glow anyway?

This seems redundant if that PR goes through.

**Comment by ChangelingRain** [comment]
Chasms themselves glowing doesn't make sense, they're really deep holes.

**Comment by Steelpoint** [comment]
Implication of magma far below.

I dunno but I dislike arbitrarily surrounding a chasm with a ring of lava that sprints forth from nowhere just for the sake of balance. With the chasm there's a clear reason why it appears,

I think just having the chasm have a dull red glow coming from it would be sufficient. Put more of the onus on the player paying attention to not run into the light.

**Comment by GunHog** [comment]
The other PR was to make the _tendril_ glow.

**Comment by Repukan** [comment]
We're talking about making the chasm glow, not leave a glowing tendril

**Comment by ChangelingRain** [comment]
#27479 but kor didn't like it since apparently chasms aren't actually a barrier so you could kill the tendril and get the chest without much risk...

**Comment by Repukan** [comment]
And, you know, the tendril remaining looked goody as fuck

**Comment by adamkad1** [comment]
Looks pretty good to me, but im not lifeweb level fun hater like some others

**Comment by AlexCrimson** [comment]
I am not opposed to this because lagging into a chasm is never fun. That said, making things easier is actually a serious issue, in my opinion. Hate to use the word, because people treat it like a joke now, but "hugbox" comes to mind. 

Lifeweb is the opposite end of the spectrum, but its the best example of why making things easier/safer isnt necessarily making the game better or more fun.

**Comment by KorPhaeron** [comment]
Instant death isnt fun but never dying is even less fun, which is why people flock to lavaland in the first place. If you remove every "unfair" thing (monsters spawn close, chasms are instant death, weapons feel a bit weak) people will just be bored instead

**Comment by KorPhaeron** [comment]
The rest of the jobs already have padded corners and this job is more popular than them, so why emulate those jobs?

**Comment by XDTM** [comment]
There are other factors that make mining attractive: lots of unique loot, almost no player interaction required, possibility of self-antag, not necessarily because you can die more often.

Also there's death from losing a fight against megafauna and there's death from stepping on a tile you forgot was there because the light hadn't caught up with you yet. The first one is fun because it was your fault, you know you can improve and avoid it the next time. The last one is not fun, since you will only ever die from it due to external factors, usually lag.

We can keep mining hard and risky without resorting to "unfair" things.

**Comment by XDTM** [comment]
That said, i like the glowing pits better.

**Pull request closed without merging**